lib/gcp/runtimeconfig.js

```diff
@@ -8,10 +8,28 @@ var _ = require('lodash');
 
 var API_VERSION = 'v1beta1';
 
+function _retryOnServerError(requestFunction) {
```

Comment:
An argument could be made that this should be baked into the `api` library instead of added as a one-off here. Since we're not using it elsewhere yet, it's probably fine, but let's make sure we don't re-write the same code for another use case and do the proper abstraction then.